src/cephadm/cephadm

```diff
@@ -3201,6 +3217,13 @@ def _parse_args(av):
 
     return args
 
+def _get_arg(key, default=None):
```

Comment:
why not just use args.retry and args.timeout and have them default in the argparse definition to 10 and 30 (or whatever)?

Reply:
hmm yep the extra indirection via the func really isn't needed or useful